targets/LumberyardSdk/make.js

```diff
@@ -751,7 +751,7 @@ function GetAuthParams(apiCall) {
     if (apiCall.auth === "SecretKey")
         return "\"X-SecretKey\", PlayFabSettings::playFabSettings->developerSecretKey";
     else if (apiCall.auth === "SessionTicket")
-        return "\"X-Authorization\", *mUserSessionTicket";
+        return "\"X-Authorization\", mUserSessionTicket";
```

Comment:
Aha, this one returns.  I'm confused about this one.
You guys are the ones who requested it become a pointer, and not a static.
I thought the previous change was a typo, because it didn't appear to be complete (it didn't change all uses).

Could you guys please confirm whether you actually intend to change this to a static?